easybuild/framework/easyblock.py

```diff
@@ -2405,7 +2405,7 @@ def post_install_step(self):
             lib_dir = os.path.join(self.installdir, 'lib')
             lib64_dir = os.path.join(self.installdir, 'lib64')
             if os.path.exists(lib_dir) and not os.path.exists(lib64_dir):
-                symlink(lib_dir, lib64_dir)
+                symlink('lib', lib64_dir, use_abspath_source=False)
```

Comment:
```suggestion
                # create *relative* 'lib64' symlink to 'lib';
                # see https://github.com/easybuilders/easybuild-framework/issues/3564
                symlink('lib', lib64_dir, use_abspath_source=False)
```

Reply:
FTR: I'm not a big fan of such comments: The first is superflous as it states the same as `use_abspath_source=False` and if the second (mentioning issues) becomes a habit the code might be cluttered by those in the future.
Added anyway, just to raise awereness